packages/esm-patient-chart-app/src/root.component.tsx

```diff
@@ -37,6 +37,7 @@ export default function Root() {
 /**
  * DO NOT REMOVE THIS COMMENT
  * THE TRANSLATION KEYS AND VALUES USED IN THE COMMON LIB IS WRITTEN HERE
+ * t('paginationPageText', 'of {count} pages', {count})
```

Comment:
```suggestion
 * The following are translation keys and strings for components in `esm-patient-common-lib`
 * t('paginationPageText', 'of {total} pages')
```

Reply:
Hi @denniskigen !
The `count` variable is used for pluralization purpose, and hence it was renamed.